### ISSUE TITLE
gap_fit error during compile (make)

### ISSUE THREAD

**Issue by DGKang234** [libAtoms/QUIP#257]
Hello sirs,

I attempted to compile the QUIP with GAP code on my account (added GAP dir in ~/QUIP/src,  HAVE_GAP=1, QUIPPY_INSTALL_OPTS=--user)

However the code abort the comiling process with the error after "**make install-quippy**":
```
>>> import quippy
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/uccatka/.local/lib/python3.7/site-packages/quippy-https_github.com_libAtoms_QUIP.git_357e28d15_dirty-py3.7-linux-x86_64.egg/quippy/__init__.py", line 3, in <module>
    import f90wrap.runtime
  File "/home/uccatka/.local/lib/python3.7/site-packages/f90wrap/runtime.py", line 33, in <module>
    from f90wrap.arraydata import get_array
ImportError: /home/uccatka/.local/lib/python3.7/site-packages/f90wrap/arraydata.cpython-37m-x86_64-linux-gnu.so: undefined symbol: __intel_sse2_strcat
```

Could you please give advice to solve the problem?

Thank you in advance.

**Comment by DGKang234** [comment]
I have contacted the rc support team to enquire about the issue.

They said it means that I installed quippy with the **Intel compiler** loaded and **not** the **gnu compiler**, and our Pythons are all built with **gnu.**

I have to switch compilers before I install Python packages. Then I can run them without switching, 
but I need to build them with the gnu compiler if they have any underlying Fortran or C.
